Description: Build node_g without libv8 debug version
 libv8 package does not provide it.
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2010-01-01
--- a/wscript
+++ b/wscript
@@ -435,7 +435,7 @@
       conf.fatal("Cannot find v8")
 
     if o.debug:
-      if not conf.check_cxx(lib=o.shared_v8_libname + '_g', header_name='v8.h',
+      if not conf.check_cxx(lib=o.shared_v8_libname, header_name='v8.h',
                             uselib_store='V8_G',
                             includes=v8_includes,
                             libpath=v8_libpath):
